app/__init__.py

```diff
@@ -12,23 +12,33 @@
 
 def create_app(test_config=None):
     app = Flask(__name__)
-    app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 
-    if test_config is None:
-        app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
+    if not test_config:
+        app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+        app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get(
             "SQLALCHEMY_DATABASE_URI")
+            
     else:
         app.config["TESTING"] = True
+        app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
         app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
             "SQLALCHEMY_TEST_DATABASE_URI")
+    app.config['JSON_SORT_KEYS']=False 
```

Comment:
Nice! Flask automatically organizes keys alphabetically so adding this line ensures json are kept in the order you provided. This isn't actually recommended by Flask for performance reasons, but it is a feature.

More info can be found in documentation: [https://flask.palletsprojects.com/en/2.0.x/config/#JSON_SORT_KEYS](https://flask.palletsprojects.com/en/2.0.x/config/#JSON_SORT_KEYS)

I also found this [stackoverflow answer](https://stackoverflow.com/questions/54446080/how-to-keep-order-of-sorted-dictionary-passed-to-jsonify-function) helpful.